### ISSUE TITLE
Final Tweak: Punctuation and Capitalization 

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#135]
Due to the fact that VO reads out `aria-valuetext` first and then adds it's own punctuation (usually a comma) before reading the label name and function name, @terracoda thinks it is best to remove capitalization and periods from strings that appear in `aria-valuetext`. ARIA-live alerts are more independent, so when crucial, it we could consider managing sentence capitalization and punctuation more directly for alerts only. 

@terracoda will attempt to make these changes.
Things to look for
- [x] periods at end of `aria-valuetext` as they are not truly complete sentences
- [x] remove capitalization on "Left/Right side of track" as it only shows up in `aria-valutext`
- [ ] with the above in mind, the only alert that won't have sentence case is the state-info alert when a user tries to go beyond and edge position. 

At this point, item 3 is not critical. We can look more closely at this as we finish off GFL Regular.

**Comment by terracoda** [comment]
@zepumph, I think https://github.com/phetsims/inverse-square-law-common/commit/f93f99d933cd7975f37575e3a02e735f98ecdb98 addresses this issue.

Please review, locally, I still see bracketed parameter in some alerts, but that shouldn't be related to my changes.

**Comment by zepumph** [comment]
Those changes look great. I linked the commit to this issue via a comment on the commit. Looks like you linked with `#{number}` which will just link to the issue in the repo fo that commit, so in this case phetsims/inverse-square-law#135 (which doesn't exist). You can link it directly in the commit message across repos by adding something like `phetsims/graivty-force-lab-basics#135` to the end of the commit (or actually the whole url to the issue work too, like `https://github.com/phetsims/gravity-force-lab-basics/issues/135`. 

>Please review, locally, I still see bracketed parameter in some alerts, but that shouldn't be related to my changes.

Those should be fixed now, please create a new issue if you see any of those leaking into the sim at this point.

>with the above in mind, the only alert that won't have sentence case is the state-info alert when a user tries to go beyond and edge position.

How would you like to proceed with this piece? During our meeting you had suggested potentially re-ordering, or I can also duplicate all the regions as capitalized words if that is best.

Over to you.

**Comment by terracoda** [comment]
Regarding, the third checkbox, I will just play a little and then comment.
Will re-assign then.

**Comment by terracoda** [comment]
@zepumph, I would indeed like to have these "going-beyond-the-edge" alerts to have proper sentence case with a capital letter at the start. 

In addition, could you append the **other** mass's name to the described distances, so they look like this:

**Random examples for mass 1:**
- **F**arthest **from mass 2**, force arrows tiny, forces 0.7 ‪newtons‬. 
- **C**losest **to mass 2**, force arrows tiny, forces 39.5 ‪newtons‬.
- **E**xtremely close **to mass 2**, force arrows very small, forces 144.2 ‪newtons‬.
- **V**ery close **to mass 2**, force arrows huge, forces 1067.9 ‪newtons‬.

**Random examples for mass 2:**
- **E**xtremely close **to mass 1**, force arrows small, forces 184.7 ‪newtons‬.
- **F**ar **from mass 1**, force arrows tiny, forces 11.5 ‪newtons‬.
- **V**ery close **to mass 1**, force arrows large, forces 454 ‪newtons‬.
- **F**arthest **from mass 1**, force arrows tiny, forces 21.7 ‪newtons‬.

I imagine you already have logic to handle the addition of the other mass's name and the addition of the appropriate preposition, but if you don't we can leave the addition of the **other** mass's name for now and make this change when we work on Gravity Force Lab Regular.

**Comment by terracoda** [comment]
@zepumph, since this is a new request, I think it requires a new issue. I will do that.

In this issue, just capitalize the alerts ;-) I moving my last comment to a new issue.

**Random examples for mass 1:**
- **F**arthest, force arrows tiny, forces 0.7 ‪newtons‬. 
- **C**losest, force arrows tiny, forces 39.5 ‪newtons‬.
- **E**xtremely close, force arrows very small, forces 144.2 ‪newtons‬.
- **V**ery close, force arrows huge, forces 1067.9 ‪newtons‬.

**Random examples for mass 2:**
- **E**xtremely close, force arrows small, forces 184.7 ‪newtons‬.
- **F**ar, force arrows tiny, forces 11.5 ‪newtons‬.
- **V**ery close*, force arrows large, forces 454 ‪newtons‬.
- **F**arthest, force arrows tiny, forces 21.7 ‪newtons‬.

**Comment by zepumph** [comment]
This capitalization was actually easier than I thought. The regions in question weren't used anywhere else, so I could just capitalized them in line, and didn't need to create a new `DISTANCE_REGIONS_CAPITALIZED` list or something like that. @terracoda please review.

**Comment by terracoda** [comment]
Capitalization on the alerts great. Thanks @zepumph. Closing.
